### DISCUSSION TITLE
Searcheable Select messes up the ordering of items

### DISCUSSION THREAD

**Discussion by mpestanasoares** [filamentphp/filament#4117] [General]
I'm using the Select component on a form. I noticed that making it searchable makes the items always sorted by value. Is there any way to do the alphabetical sorting?

For example, if I use the options below:

```
Forms\Components\Select::make('question1')
                            ->options([
                    '110'=>'Alice', 
                    '20'=>'Bernard', 
                    '1'=>'Cathy', 
                    '3'=>'Damian',
                    '10'=>'James',
                    '2' =>'John', 
                    '300'=>'Lucas', 
                    '64'=>'Maria', 
                    '9'=>'Zeus'])
                            ->searchable();
```
That's the result:
<img width="180" alt="select_filament" src="https://user-images.githubusercontent.com/72081942/191387457-e4dfad19-d258-48b9-86f2-7fa454a7bfa8.png">

**Comment by danharrin** [comment]
Hey, we fixed this several days ago. Please upgrade to the latest version, clear your view cache, and recompile your assets.

**Reply by mpestanasoares** [reply]
It worked!
